generate/report_test.go

```diff
@@ -329,7 +329,13 @@ func assertEqualReports(t *testing.T, got, want generate.Report) {
 
 	if diff := cmp.Diff(got.Successes, want.Successes); diff != "" {
 		t.Errorf("success results differs: got(-) want(+)")
-		t.Fatal(diff)
+		// Don't call fatal to avoid vague errors like
```

Comment:
Don't call fatal to avoid vague errors like:

```
generate_test.go:61: success results differs: got(-) want(+)
generate_test.go:61:   []generate.Result(
		-   nil,
		+   {{Dir: s"/stacks/stack", Created: []string{"dir/file.hcl", "dir/sub/file.txt"}}},
)
```
With no info on what was the error/failure.